pymchelper/executor/options.py

```diff
@@ -43,13 +51,24 @@ class SimulationSettings:
     (i.e. whether this is SHIELD-HIT12A input or FLUKA input)
     """
 
-    def __init__(self, input_path, simulator_exec_path=None, cmdline_opts=None):
+    def __init__(self, input_path, simulator_type=SimulatorType.shieldhit, simulator_exec_path=None, cmdline_opts=None):
```

Comment:
```suggestion
    def __init__(self, input_path: str, simulator_type=SimulatorType.shieldhit, simulator_exec_path=None, cmdline_opts=None):
```

do we handle input_path as `Path` or `str` ? I suspect that `str`

Reply:
str